donald_wade/week_04/wednesday/film_api/film_api.rb

```diff
@@ -0,0 +1,42 @@
+require 'sinatra'
+require 'sinatra/reloader'
+require 'sinatra/content_for'
+require 'httparty'
+require 'json'
+require 'pp'
+
+get '/' do
+  url_string = ""
+  pp params
+  pp params["your_search"]
+    if params["your_search"] != ""
+      @your_search_request = params["your_search"]
+      if @your_search_request != nil
+        @big_result = HTTParty.get("http://www.omdbapi.com/", query:{ s: @your_search_request})
```

Comment:
what happens if you search for movie that has more then 1 word? For example "Home alone"?